packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/table-of-contents-hierarchy.md

```diff
@@ -1,5 +1,5 @@
 ### Description
-This Pattern shows an expantable table of contents which can contain simple links, download links, or expandable sections.
+This Pattern shows an expandable table of contents which can contain simple links, download links, or expandable sections.
```

Comment:
Can you update this doc to include the new configuration? 
https://mayflower.digital.mass.gov/patternlab/b/patternlab/DP-22640_topic-page-redesign-fe/?p=organisms-table-of-contents-hierarchy-no-colored
"background" : "none",

Reply:
done.